### PR TITLE
DOCSP-14071: use stream() on cursor for streaming api

### PR DESCRIPTION
**Pull request by terakilobyte**
## Pull Request Info

Per the ticket description in mongodb@4, `stream()` must now be called on the cursor to use the stream API.

### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-14071

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/faaae47/node/docsworker-xlarge/DOCSP-14071/fundamentals/crud/read-operations/cursor/#stream-api

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [x] Are all the links working?
- [x] Are the staging links in the PR description updated?

### CODE REVIEW

**Comment by terakilobyte** [comment]
@nbbeeken @emadum 

I've updated the stream example per the engineering ticket, and clarified the text. I also removed the pipe operation and just used events on the Readable, as users who want to use this API will probably be familiar with it.

**Comment by nbbeeken** [comment]
Do we have a way of keeping multiple versions? Kind of like the language pills on other docs pages?
The 4.x version of the driver is still in beta, we're doing the RC this month but the official release is still unscheduled. I'm wondering if we want to hold off merging this or provide a way to show both methods.

Of course, I defer to what your thoughts are on this. Maybe it is overkill to maintain both since we have plenty of docs on our old page to cover the 3.x version.

**Comment by terakilobyte** [comment]
We'll ensure a 3.6 branch  is cut. It's on the docket for next sprint work.

**Review by nbbeeken — APPROVED** [review]
LGTM

**Review by emadum — CHANGES_REQUESTED** [review]
Oops never mind.

**Review by emadum — APPROVED** [review]

**Pull request merged**
